internal/monitors/collectd/postgresql/postgresql.tmpl

```diff
@@ -0,0 +1,67 @@
+LoadPlugin postgresql
+
+<Plugin postgresql>
+  <Query custom_deadlocks>
+    Statement "SELECT deadlocks as num_deadlocks \
+        FROM pg_stat_database \
+        WHERE datname = $1;"
+    Param database
+    <Result>
+      Type "pg_xact"
+      InstancePrefix "num_deadlocks"
+      ValuesFrom "num_deadlocks"
+    </Result>
+  </Query>
+  {{range $q := .Queries}}
+  <Query "{{$q.Name}}">
+    Statement "{{$q.Statement}}"
+    {{range $param := $q.Params -}}
+    Param "{{$param}}"
+    {{- end}}
+    {{with $q.PluginInstanceFrom}}PluginInstanceFrom "{{.}}"{{- end}}
+    {{with $q.MinVersion}}MinVersion {{.}}{{- end}}
+    {{with $q.MaxVersion}}MaxVersion {{.}}{{- end}}
+    {{range $r := $q.Results -}}
+    <Result>
+      Type "{{$r.Type}}"
+      {{with $r.InstancePrefix -}}InstancePrefix "{{.}}"{{- end}}
+      {{if $r.InstancesFrom -}}InstancesFrom {{range $from := $r.InstancesFrom}}"{{$from}}" {{end}}{{- end}}
+      ValuesFrom {{range $v := $r.ValuesFrom}}"{{$v}}" {{- end}}
+    </Result>
+    {{- end}}
+  </Query>
+  {{end}}
+  {{range $db := .Databases}}
+  <Database "{{$db.Name}}">
+    Host "{{$.Host}}"
+    Port "{{$.Port}}"
+    ReportHost {{$.ReportHost}}
```

Comment:
Use `{{toBool $.ReportHost}}` instead of relying on the stringified form of golang bools in the templating engine.